management/status_checks.py

```diff
@@ -965,7 +965,11 @@ def run_and_output_changes(env, pool):
 	cache_fn = "/var/cache/mailinabox/status_checks.json"
 	if os.path.exists(cache_fn):
 		with open(cache_fn, 'r') as f:
-			prev = json.load(f)
+			try:
+				prev = json.load(f)
+			except json.JSONDecodeError:
+				logging.debug('Could not decode previous status checks JSON file')
```

Comment:
I don't think `logging` is defined in this module.

Reply:
oooh, busted 😅 
I'll remove that line.